src/docker-osx-dev

```diff
@@ -399,7 +399,7 @@ function configure_docker_machine {
   DOCKER_HOST_NAME="$DOCKER_MACHINE_NAME"
   DOCKER_HOST_USER=$(inspect_docker_machine "{{.Driver.SSHUser}}")
   DOCKER_HOST_IP=$(inspect_docker_machine "{{.Driver.IPAddress}}")
-  DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.StorePath}}")
+  DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.HostOptions.AuthOptions.StorePath}}")
```

Comment:
Thanks for looking this up.

Assuming it doesn't require a ton of hard-to-maintain code, I'd prefer to keep docker-osx-dev backwards compatible, where possible. It's painful enough to upgrade Docker versions as it is, so I'd rather not add to that upgrade headache by having to worry about docker-osx-dev versions as well.